design/baremetal-operator/image-builder-integration.md

```diff
@@ -7,6 +7,8 @@
 
 # Custom agent image controller
 
+<!-- cSpell:ignore statelessly -->
```

Comment:
nit: maybe we should use "in a stateless way" or something like that

Reply:
Sure, I did not want to do editorials in such a big PR, I can fix that in follow-up.